.circleci/config.yml

```diff
@@ -0,0 +1,56 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/desktop/desktop
+    parallelism: 1
+    shell: /bin/bash --login
+    environment:
+      CIRCLE_ARTIFACTS: /tmp/circleci-artifacts
+      CIRCLE_TEST_REPORTS: /tmp/circleci-test-results
+    macos:
+      xcode: '9.3.0'
+    steps:
+    - checkout
+    - run: mkdir -p $CIRCLE_ARTIFACTS $CIRCLE_TEST_REPORTS
+    - restore_cache:
+        keys:
+        # when lock file changes, use increasingly general patterns to restore cache
+        - yarn-packages-v1-{{ .Branch }}-{{ checksum "yarn.lock" }}
+        - yarn-packages-v1-{{ .Branch }}-
+        - yarn-packages-v1-
+    - run:
+        name: Install dependencies
+        command: yarn install --force
```

Comment:
I kept the `--force` flag from the 1.0 config, but this essentially renders the caching using `yarn.lock` useless. @shiftkey do we want to keep this?

Reply:
I'm fine with keeping this around. Reading the [docs](https://yarnpkg.com/en/docs/cli/install) again, I read the use of `--force` as being related to the downloading of packages, ignoring the cache, rather than ignoring the lockfile (`--no-lockfile`):

<img width="613" src="https://user-images.githubusercontent.com/359239/43339671-5dd08fac-91b0-11e8-9ea2-a33284e7e910.png">
<img width="468" src="https://user-images.githubusercontent.com/359239/43339673-5de3b74e-91b0-11e8-826e-e053f5ce4f13.png">

Let me know if you disagree.

Reply:
I was under the impression that we're caching `node_modules` (line 32 just below) but `--force` would reinstall all deps even if they're already in `node_modules`.

Reply:
You are correct, and I think we brought this in flag because we had problems building when the Electron version would change underneath (caching `node_modules` meant the build would break with an obscure `EEXISTS` error). I'll dig into this on Monday and report back.

Reply:
Okay! If we end up deciding `--force` flag is necessary to keep, I'd skip caching for `node_modules` since it's an expensive operation and won't be doing anything.

Reply:
:thumbsup: caching this folder requires rehydrating the folder for each build, which is wasted cycles if we ignore the cache anyway

Reply:
For the moment I want to keep this old behaviour around, just in case it's helpful for us down the track. I had a chat with @shana a few days ago about [offline mirroring into the repository](https://yarnpkg.com/blog/2016/11/24/offline-mirror/) which commits the packages (not the `node_modules` contents) and would be a great speedup for CI scenarios like this.

I'll open an issue to track this work once I've done some initial scouting.